libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ptr/CustomSwipeRefreshLayout.java

```diff
@@ -0,0 +1,29 @@
+package org.wordpress.android.util.ptr;
+
+import android.content.Context;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+
+import org.wordpress.android.util.AppLog;
+import org.wordpress.android.util.AppLog.T;
+
+public class CustomSwipeRefreshLayout extends SwipeRefreshLayout {
+    public CustomSwipeRefreshLayout(Context context) {
+        super(context);
+    }
+
+    public CustomSwipeRefreshLayout(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    @Override
+    public boolean onTouchEvent(MotionEvent event) {
```

Comment:
This could use a comment explaining why it's necessary, perhaps with a link to the [platform fix](https://github.com/android/platform_frameworks_support/blob/5476e7f4203acde2b2abbee4e9ffebeb94bcf040/v4/java/android/support/v4/widget/SwipeRefreshLayout.java#L673-L811) for reference.